docs/drivers/drivers.md

```diff
@@ -0,0 +1,59 @@
+## Drivers Maintenance 
+
+#### How to upgrade the Nvidia drivers in your cluster
+
+
+###### Available GPU drivers version in OpenPAI
+
+```
+"384.111"  (Default Value)
+```
+
+
+###### Cluster Configuration
+
+- Choose an available version in Openpai, and change the ```drivers``` section in ```service-configuration.yaml```.  
+
+- Update the configuration in your cluster with the command ```paictl config push```. If you wanna investigate more detail about this command, please refer to this [link](../paictl/paictl-manual.md#Config_Push) 
+
+###### Stop corresponding service
+
+- Delete hadoop-node-manager and delete all jobs
+    - Note: all jobs will be killed
+       
```

Comment:
I think `killed` is a little confusing, actually these job will retry when node-manager is available.

Reply:
Add more words? Such as `all running jobs will be killed and retry after upgrading`

Reply:
Done